example/spec/parallel/testAPI/sqsRuleSpec.js

```diff
@@ -238,5 +244,14 @@ describe('The SQS rule', () => {
     it('messages are picked up and removed from source queue', async () => {
       await expectAsync(waitForQueueMessageCount(queues.sourceQueueUrl, 0)).toBeResolved();
     });
+
+    it('stores incoming messages on S3', async () => {
+      const keyOne = `${config.stackName}/archived-incoming-messages/${messageId}`;
```

Comment:
We could use the helper to get this key, now that we have it

Reply:
Does this require publishing the ingest package again? @markdboyd